rasa/core/policies/ted_policy.py

```diff
@@ -591,6 +646,42 @@ def train(
             shuffle=False,  # we use custom shuffle inside data generator
         )
 
+    def train(
+        self,
+        training_trackers: List[TrackerWithCachedStates],
+        domain: Domain,
+        interpreter: NaturalLanguageInterpreter,
+        **kwargs: Any,
+    ) -> None:
+        """Trains the policy on given training trackers.
+
+        Args:
+            training_trackers: List of training trackers to be used
+                for training the model.
+            domain: Domain of the assistant.
+            interpreter: NLU Interpreter to be used for featurizing the states.
+            **kwargs: Any other argument.
+        """
+        if not training_trackers:
+            logger.error(
```

Comment:
is this rather a warning? What can users do to fix it?

Reply:
Hmm, didn't give it a thought because it was always like this but yes warning seems more appropriate. Changing it now.

> What can users do to fix it?

They should ideally not have this policy in the config as it is untrainable as a result of which it will return default predictions during inference which will not affect ensemble at all. Will add a note for this in the message.

Reply:
Thanks!